AmplifyPlugins/Predictions/AWSPredictionsPlugin/Service/Predictions/AWSPredictionsService+Transcribe.swift

```diff
@@ -31,7 +31,16 @@ extension AWSPredictionsService: AWSTranscribeStreamingServiceBehavior {
         request.mediaSampleRateHertz = 8_000
 
         transcribeClientDelegate.connectionStatusCallback = { status, error in
-            if status == .connected {
+            if status == .closed && error != nil {
+                let nsError = error as NSError?
+                if nsError?.domain == NSURLErrorDomain {
```

Comment:
by working off of URLError (https://developer.apple.com/documentation/foundation/urlerror) you get an object with additional information

Reply:
```swift
if status == .closed  {
   guard error != nil else {
     return
   }
  guard errorIsNotNormalClosedError(error) else {
     return
  }
   let predictionsError = PredictionsHelper.mapError(nsError)
   onEvent(.failed(predictionsError)
  
} else if status == .connected {

}

```